libs/datasets/sources/cds_dataset.py

```diff
@@ -111,21 +124,27 @@ def standardize_data(cls, data: pd.DataFrame) -> pd.DataFrame:
         # The following abbrev mapping only makes sense for the US
         # TODO: Fix all missing cases
         data = data[data["country"] == "United States"]
-        data["state_abbr"] = data[cls.Fields.STATE].apply(
+        data[CommonFields.COUNTRY] = "USA"
+        data[CommonFields.STATE] = data[cls.Fields.STATE].apply(
             lambda x: US_STATE_ABBREV[x] if x in US_STATE_ABBREV else x
         )
-        data["state_tmp"] = data["state"]
-        data["state"] = data["state_abbr"]
 
         fips_data = dataset_utils.build_fips_data_frame()
         data = dataset_utils.add_fips_using_county(data, fips_data)
+        no_fips = data[CommonFields.FIPS].isna()
+        if no_fips.sum() > 0:
+            logging.warning(f"Removing rows without fips id: {str(data.loc[no_fips])}")
+            data = data.loc[~no_fips]
+
+        data.set_index(["date", "fips"], inplace=True)
+        if data.index.has_duplicates:
+            logging.warning(f"Removing duplicates: {str(data.index.duplicated(keep=False))}")
+            data = data.loc[~data.index.duplicated(keep=False)]
```

Comment:
What does keep=False do?

Reply:
keep=False makes the log message easier to read because it includes all of the duplicated row instead of just the first or last instance of each duplicate.